src/test/java/uk/gov/hmcts/divorce/caseworker/service/task/SetPostIssueStateTest.java

```diff
@@ -0,0 +1,76 @@
+package uk.gov.hmcts.divorce.caseworker.service.task;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.divorce.divorcecase.model.Application;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.ccd.sdk.type.YesOrNo.YES;
+import static uk.gov.hmcts.divorce.divorcecase.model.ServiceMethod.COURT_SERVICE;
+import static uk.gov.hmcts.divorce.divorcecase.model.ServiceMethod.SOLICITOR_SERVICE;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingAos;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.AwaitingService;
+import static uk.gov.hmcts.divorce.divorcecase.model.State.Holding;
+
+@ExtendWith(MockitoExtension.class)
+class SetPostIssueStateTest {
+
+    @InjectMocks
+    private SetPostIssueState setPostIssueState;
+
+    @Test
+    void shouldSetStateToAwaitingService() {
```

Comment:
would be nice to add when part `shouldSetStateToAwaitingServiceWhenServiceMethodIsSolicitor`

Reply:
Updated with 
https://github.com/hmcts/nfdiv-case-api/pull/676/commits/9b633d2b8e27db70f64207078bdd8b590c8f1f74